src/ontology/mondo-ingest.Makefile

```diff
@@ -369,3 +368,36 @@ slurp-%:
 	$(MAKE) slurp/$*.tsv
 
 slurp-all: slurp-omim slurp-doid slurp-ordo slurp-icd10cm slurp-icd10who slurp-ncit
+
+#############################
+###### Synchronization ######
+#############################
+sync/:
+	mkdir -p $@
+
+# TODO: @Nico
+tmp/mondo-ingest.db:
+	touch tmp/mondo-ingest.db
+tmp/mondo-ingest-without-excluded.db: tmp/mondo-ingest.db
+	cp $< $@
+
+# todo: `pip install` stuff is temporarily here until we come up with a fix. otherwise docker won't work
+sync/mondo-sync-%.owl: tmp/mondo.owl tmp/mondo-ingest-without-excluded.db $(TMPDIR)/mondo.sssom.tsv $(COMPONENTSDIR)/%.owl | sync/
```

Comment:
@matentzn `sync/mondo-sync-%.owl`
1. You may want to look over the look at the params and check that they look correct. May or may not need `--onto-config-path`.
2. I know that for this firs step I'll need `outpath-synonyms`. But at the end of all 5 steps, do I also want create/update to a Mondo `.owl` file?

Reply:
I don't remember why we had this initial design, but I'm pretty sure we don't need it now. We're passing individual `.db` files for each source as input, and output should be ROBOT templates.